lambda/custom/handlers/General/YesIntentHandler.js

```diff
@@ -1,34 +1,86 @@
 const { ri } = require('@jargon/alexa-skill-sdk');
+const { login, postMessage } = require('../../helperFunctions');
+const { supportsAPL } = require('../../utils');
+const titleMessageBoxTemplate = require('../../APL/templates/titleMessageBoxTemplate');
 
 const YesIntentHandler = {
 	canHandle(handlerInput) {
 		return handlerInput.requestEnvelope.request.type === 'IntentRequest' &&
 			handlerInput.requestEnvelope.request.intent.name === 'AMAZON.YesIntent';
 	},
-	handle(handlerInput) {
-
+	async handle(handlerInput) {
 		const { attributesManager } = handlerInput;
 		const sessionAttributes = attributesManager.getSessionAttributes() || {};
 
-		return handlerInput.jrb
-			.speak(ri('POST_MESSAGE.CONFIRM_MORE'))
-			.reprompt(ri('POST_MESSAGE.CONFIRM_MORE_REPROMPT'))
-			.addElicitSlotDirective('longmessage', {
-				name: 'PostLongMessageIntent',
-				confirmationStatus: 'NONE',
-				slots: {
-					channelname: {
-						name: 'channelname',
-						value: sessionAttributes.channelName,
-						confirmationStatus: 'NONE',
-					},
-					longmessage: {
-						name: 'longmessage',
-						confirmationStatus: 'NONE',
+		if (sessionAttributes.postLongMessageIntentOnProgress) {
```

Comment:
else condition is missing

Reply:
Added a generic yes prompt in else condition.
**Alexa**: Anything else I can help you with?
**User**: Yes
**Alexa**: What would you like to do?